core/src/main/java/com/opentable/logging/CommonLogFields.java

```diff
@@ -86,18 +76,24 @@ default String getOtEnvFlavor() {
     @JsonProperty("severity")
     String getSeverity();
 
-    @JsonProperty("logmessage")
+    @JsonProperty("message")
     String getMessage();
 
-    @JsonProperty("threadname")
+    @JsonProperty("thread-name")
     String getThreadName();
 
-    @JsonProperty("throwable")
+    @JsonProperty("exception")
     String getThrowable();
 
     /** Written by the encoder, value is ignored for serialization. */
-    @JsonProperty("sequencenumber")
+    @JsonProperty("sequence-number")
     default long getSequenceNumber() {
         return Long.MIN_VALUE;
     }
+
+    /** Declare the OTL type for selectorless processing. */
+    @JsonProperty("@loglov3-otl")
+    default String getLoglov3Otl() {
```

Comment:
I suggest just defining the field here and moving the `return "msg-v1";` to the implementing class `ApplicationLogEvent`.

If someone implements this interface and then adds their own fields, then they should ideally create a new OTL defining those fields. With the default method like this they would be be getting the `msg-v1` OTL. Masking the implementor choose the OTL is a subtle hint that they should create one.